services/auth/src/gcp.js

```diff
@@ -3,14 +3,16 @@ const {google} = require('googleapis');
 
 builder.declare({
   method: 'get',
-  route: '/gcp/credentials/:serviceAccount',
+  route: '/gcp/credentials/:projectId/:serviceAccount',
   name: 'gcpCredentials',
   output: 'gcp-credentials-response.yml',
   stability: 'stable',
-  scopes: 'auth:gcp:access-token:<serviceAccount>',
+  scopes: 'auth:gcp:access-token:<projectId>/<serviceAccount>',
   title: 'Get Temporary Read/Write GCP Credentials',
   description: [
     'Get temporary GCP credentials for the given serviceAccount.',
+    'projectId must always be the string "-", which means "use the same',
+    'projectId as the account the service is running at.',
```

Comment:
I think this should be the *actual* project ID, not `-`.  This comment is a bit misleading, anyway -- in Heroku, the service isn't running in any project (it's running in the Mozilla "account" but that's a different thing).  And when deployed in GKE, the auth service may be running in one project but have configuration for a different project.

Reply:
The API endpoint from Google doesn't work if the `projectId` is not "-". I believe it is a bug.

Reply:
That sort of makes sense -- the gcpCredentials are specific to that projectId, so in essence "-" means "use the projectId in the credentials".  If we were to eventually support multiple projects, we'd need to be configured with multiple sets of credentials, too.  At any rate, I don't think it prohibits changing the API to accept a projectId.